packages/messaging-system/src/components/ChatList/ChatList.module.scss

```diff
@@ -0,0 +1,39 @@
+.container {
+  height: inherit;
+}
+
+.title {
+  font-family: "Poppins", sans-serif;
+  font-size: 18px;
+  font-weight: 600;
+  color: #000000;
+}
+
+.list {
+  height: inherit;
+  width: 100%;
+  margin-top: 32px;
+  overflow-y: auto;
+}
+
+.listItemContainer {
+  width: 100%;
+  list-style: none;
+  margin-bottom: 24px;
+}
+
+.listItem {
+  background: none;
+  color: inherit;
+	border: none;
```

Comment:
I don't know what happen here, but I'll fix it on my next PR

Reply:
What is wrong here?

Reply:
indentation

Reply:
So weird, in vscode everything seems fine.
But yes, it is a just weird ident.